app/controllers/rents_controller.rb

```diff
@@ -0,0 +1,23 @@
+class RentsController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    render_paginated current_user.rents
+  end
+
+  def create
+    @rent = Rent.new(permitted_params)
+    if @rent.save
+      render json: @rent, status: :created
+    else
+      render json: { errors: @rent.errors.messages }, status: :unprocessable_entity
+    end
+  end
+
```

Comment:
you could put a private separator to differentiate the exposed endpoints from your internal logic